aea/helpers/ipfs/base.py

```diff
@@ -92,7 +92,7 @@ def get(cls, file_path: str, wrap: bool = True, cid_v1: bool = True) -> str:
         if os.path.isdir(file_path):
             return cls.hash_directory(file_path, wrap=wrap, cid_v1=cid_v1)
 
-        return cls.hash_file(file_path, wrap=wrap)
+        return cls.hash_file(file_path, wrap=wrap, cid_v1=cid_v1)
```

Comment:
this was such an obvious bug that I just decided to fix it straight up for the sake of time